spdx/writers/rdf.py

```diff
@@ -569,7 +569,7 @@ def relationships(self):
         """
         Return a list of relationship nodes
         """
-        return map(self.create_relationship_node, self.document.relationships)
+        return list(map(self.create_relationship_node, self.document.relationships))
```

Comment:
I don't think it makes a difference here whether we return an iterable or a list. The doc says it returns a list, but then again all other similar functions return an iterable as well, so I'm inclined to leave it that way (or fix all the docs)